src/main/java/com/cat2client/clientapp/Models/CreateMovieDto.java

```diff
@@ -0,0 +1,33 @@
+package com.cat2client.clientapp.Models;
+
+import java.util.List;
+
+public class CreateMovieDto {
+    private String title;
+    private List<String> categories;
+    private int runtime;
+    private List<Integer> productionCompaniesIds;
```

Comment:
Since the data required to create a movie is significantly different from the actual movie object, it is better to create a DTO (data transfer object) for the request.